bubble_sort/Python/bubbleSort.py

```diff
@@ -0,0 +1,24 @@
+#Bubble Sort in Python
+
+array = [] #declaration of array
+
+
+#getting f user inputs and append that values to the array
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmppuq6kerg/bubble_sort/Python/bubbleSort.py
+++ b/tmp/tmppuq6kerg/bubble_sort/Python/bubbleSort.py
@@ -3,7 +3,7 @@
 array = [] #declaration of array
 
 
-#getting f user inputs and append that values to the array
+# getting f user inputs and append that values to the array
 for x in range(5):
     array.append(eval(input("number " + "")))
 
```